ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-base.md

```diff
@@ -104,13 +104,15 @@ interface Management1
 
 ### BGP Neighbors
 
-| Neighbor | Remote AS | VRF | Shutdown | Send-community | Maximum-routes | Allowas-in | BFD | RIB Pre-Policy Retain |
-| -------- | --------- | --- | -------- | -------------- | -------------- | ---------- | --- | -------------- |
-| 192.0.3.1 | 65432 | default | - | all | - | - | - | True |
-| 192.0.3.2 | 65433 | default | - | extended | 10000 | - | - | True (All) |
-| 192.0.3.3 | 65434 | default | - | standard | - | - | - | True |
-| 192.0.3.4 | 65435 | default | - | large | - | - | - | False |
-| 192.0.3.5 | 65436 | default | - | standard | 12000 | - | - | - |
+| Neighbor | Remote AS | VRF | Shutdown | Send-community | Maximum-routes | Allowas-in | BFD | RIB Pre-Policy Retain | Remove Private AS Outbound | Remove Private AS Inbound |
```

Comment:
I don't think these two are used often enough to warrant two new columns on all documentation. We need to find a better way to document all this, but until then, I think we should just leave out these knobs from the tables. It is still shown in the config below.

Reply:
Removed.